amlb/runners/container.py

```diff
@@ -136,7 +135,6 @@ def _run():
                     else rconfig().seed,
                 )
             )
-            # TODO: would be nice to reload generated scores and return them
```

Comment:
While that would follow the other job results, results currently get printed and saved to logs fine? So I am not 100% sure what the added value is. Might be missing something, didn't have a close look.